### PR TITLE
create contact us form

### PR DESCRIPTION
**Pull request by saiharsha-22**
solved the issue [CREATE] Contact Us #189

### CODE REVIEW

**Comment by sujata13** [comment]
Can you please also attach the snippet of the page that you have created. You can share it here in comment section or on our [Telegram Group](https://t.me/hacktober_fest_2020).

**Comment by saiharsha-22** [comment]
@sujata13 I have added in the telegram channel before I have added once again.
WhatsApp Image 2020-10-03 at 8 41 55 PM

**Comment by sujata13** [comment]
Thanks for contribution. Happy Coding!!!

File: Day02/Contact us/contactus.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    <div class="column">
+           <h2>Call Or Email</h2>
+           <p>Support, Sales, and Account Management Services are currently
+              available in English.
+           </p>
+           <h3>Support</h3>
+           <p>+800 3005 4300</p>
+            <p class="blue">Contact Support</p>
+            <p>Our technical support is available by phone or email form 11am to 
+              11pm BST,Monady through Friday
```
Comment (Ujaan07) [inline Day02/Contact us/contactus.html:90]: Please change  monady to Monday.

File: Day02/Contact us/contactus.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+            </p> 
+            <h3>Sales</h3>
+            <p>+800 3005 4300</p>
+             <p class="blue">Contact Sales</p>
+             <p>Our technical support is available by phone or email form 11am to 
+               11pm BST,Monady through Friday
```
Comment (Ujaan07) [inline Day02/Contact us/contactus.html:96]: Here also Monday instead of Monady

**Pull request merged**